docs/how-to.md

```diff
@@ -0,0 +1,63 @@
+---
+title: How-Tos
+order: 50
+---
+
+# Repository Changes
+
+**NOTE:** Changes should be coordinated such that the access controls at
+`hg.mozilla.org` are always more restrictive than those configured in
+mozilla-taskcluster.  For example, when raising a repo from level 2 to level 3,
+make the change in `hg.mozilla.org` first.  Likewise when lowering a repo from
+level 3 to level 2, make the change in mozilla-taskcluster first. This avoids a
+situation where a user with level-2 credentials could trigger level-3 tasks.
```

Comment:
Is it for the scm level in taskcluster to diverge from that in `production-branches.json`. I'd be inclined to have comm-central be level1 or level2 in taskcluster until it is ready to tier 1. (At least partly to avoid comm-central inadvertently clobbering indexes or caches that are used by gecko level 3 builds.